lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -102,14 +101,17 @@ func (i *InteractionHandlersImpl) ContainerSetStdinHandler(params interaction.Co
 	log.Printf("Attempting to get ssh session for container %s stdin", params.ID)
 	sshConn, err := i.attachServer.Get(context.Background(), params.ID, interactionTimeout)
 	if err != nil {
-		e := &models.Error{Message: fmt.Sprintf("No stdin found for %s", params.ID)}
-		return interaction.NewContainerSetStdinNotFound().WithPayload(e)
+		err = fmt.Errorf("No stdin found (id:%s): %s", params.ID, err.Error())
+		log.Errorf("%s", err.Error())
+
+		return interaction.NewContainerSetStdinNotFound()
```

Comment:
presume the payload is dropped because it adds no value beyond the base error? (assuming the caller knows which container the connection corresponds to..)